Replace outdated instructions and add a tip in kubeadm-certs.md (#41939)
kubernetes/website: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

## Manual certificate renewal

> You can renew your certificates manually at any time with the `kubeadm certs renew` command, with the appropriate command line options.

Reviewer: we can omit this change, arguably. calling just `kubeadm certs renew` will signal to the user that they need to pick "all" or a specific target.

Author: I mean, technically, you could.  But I'd argue that the documentation should make things as easy as possible, to aid new users.

Reviewer: i prefer for us not to maintain command output - e.g. a subcommand list in the documentation

Author: That makes sense to me.

to keep them both in sync.
{{< /note >}}

> `kubeadm certs renew` can renew any specific certificate or, with the subcommand `all`, it can renew all of them, as below:

Reviewer: ```suggestion
`kubeadm certs renew` can renew any specific certificate, or with the subcommand `all`, it can renew all of them, as below:
```

Author: Thank you.

to keep them both in sync.
{{< /note >}}

> `kubeadm certs renew` can renew all certificates, using `kubeadm certs renew all` or can renew only specific certificates, as shown in the excerpt below of its command line help:

Reviewer: ```suggestion
`kubeadm certs renew` can renew any specific certificate or with a subcommand `all` it can renew all of them:
```

> ```console

Reviewer: ```suggestion
```shell
```

Author: I'd like to push back on this suggestion - the style guide demonstrates in [Separate commands from output](https://github.com/kubernetes/website/edit/main/content/en/docs/contribute/style/style-guide.md) that commands should use \`\`\`shell, while their output should use \`\`\`console.

Excerpt from the style guide follows:

```
### Separate commands from output

Verify that the pod is running on your chosen node:

```shell
kubectl get pods --output=wide
```

The output is similar to this:

```console
NAME     READY     STATUS    RESTARTS   AGE    IP           NODE
nginx    1/1       Running   0          13s    10.200.0.4   worker0
```

Author: Thank you, @neolit123!

  admin.conf               Renew the certificate embedded in the kubeconfig file for the admin to use and for kubeadm itself
  all                      Renew all available certificates
  apiserver                Renew the certificate for serving the Kubernetes API
  apiserver-etcd-client    Renew the certificate the apiserver uses to access etcd
  apiserver-kubelet-client Renew the certificate for the API server to connect to kubelet
  controller-manager.conf  Renew the certificate embedded in the kubeconfig file for the controller manager to use
  etcd-healthcheck-client  Renew the certificate for liveness probes to healthcheck etcd
  etcd-peer                Renew the certificate for etcd nodes to communicate with each other
  etcd-server              Renew the certificate for serving etcd
  front-proxy-client       Renew the certificate for the front proxy client
  scheduler.conf           Renew the certificate embedded in the kubeconfig file for the scheduler manager to use
```

{{< note >}}

> Clusters built with kubeadm often copy the `admin.conf` certificate into $HOME/.kube/config, as instructed in in [Creating a cluster with kubeadm](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/). After renewing all certificates on such a system, you may want to run these commands:

Reviewer: ```suggestion
Clusters built with kubeadm often copy the `admin.conf` certificate into `$HOME/.kube/config`, as instructed in [Creating a cluster with kubeadm](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/). On such a system, to update the contents of `$HOME/.kube/config` after renewing the `admin.conf` you must run the following commands:
```
